### ISSUE TITLE
fix(node-nest-starter-2020-04-03): frontend commands fail

### ISSUE THREAD

**Issue by mavogel** [akveo/ngx-admin-bundle-support#66]
### Problem
Commands on the `frontend` folder fail

1. testing
```sh
$ npm test

ERROR in multi ./node_modules/bootstrap/dist/css/bootstrap.css ./node_modules/typeface-exo/index.css ./node_modules/roboto-fontface/css/roboto/roboto-fontface.css ./node_modules/ionicons/scss/ionicons.scss ./node_modules/font-awesome/scss/font-awesome.scss ./node_modules/socicon/css/socicon.css ./node_modules/nebular-icons/scss/nebular-icons.scss ./node_modules/pace-js/templates/pace-theme-flash.tmpl.css ./src/app/@theme/styles/styles.scss
Module not found: Error: Can't resolve '/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss' in '/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend'
resolve '/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss' in '/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend'
  using description file: /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/package.json (relative path: ./node_modules/font-awesome/scss/font-awesome.scss)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.ts doesn't exist
      .tsx
        Field 'browser' doesn't contain a valid alias configuration
        /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.tsx doesn't exist
      .mjs
        Field 'browser' doesn't contain a valid alias configuration
        /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.mjs doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.js doesn't exist
      as directory
        /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss doesn't exist
[/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss]
[/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.ts]
[/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.tsx]
[/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.mjs]
[/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/font-awesome/scss/font-awesome.scss.js]
 @ multi ./node_modules/bootstrap/dist/css/bootstrap.css ./node_modules/typeface-exo/index.css ./node_modules/roboto-fontface/css/roboto/roboto-fontface.css ./node_modules/ionicons/scss/ionicons.scss ./node_modules/font-awesome/scss/font-awesome.scss ./node_modules/socicon/css/socicon.css ./node_modules/nebular-icons/scss/nebular-icons.scss ./node_modules/pace-js/templates/pace-theme-flash.tmpl.css ./src/app/@theme/styles/styles.scss styles[4]
```

2. linting:
```sh
$ npm run lint:ci

> ngx-admin-bundle@3.20.2020 lint:styles /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend
> stylelint ./src/**/*.scss

Error: No configuration provided for /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/src/app/pages/pages.component.scss
    at module.exports (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/stylelint/lib/utils/configurationError.js:8:27)
    at stylelint._fullExplorer.load.then.then.config (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/stylelint/lib/getConfigForFile.js:39:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 78
npm ERR! ngx-admin-bundle@3.20.2020 lint:styles: `stylelint ./src/**/*.scss`
npm ERR! Exit status 78
npm ERR! 
npm ERR! Failed at the ngx-admin-bundle@3.20.2020 lint:styles script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mavogel/.npm/_logs/2020-04-28T16_36_32_658Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 78
npm ERR! ngx-admin-bundle@3.20.2020 lint:ci: `npm run lint && npm run lint:styles`
npm ERR! Exit status 78
npm ERR! 
npm ERR! Failed at the ngx-admin-bundle@3.20.2020 lint:ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mav
```

3. e2e testing
```sh
$ npm run e2e

** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
: Compiled successfully.
(node:98827) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
[18:41:42] I/launcher - Running 1 instances of WebDriver
[18:41:42] I/direct - Using ChromeDriver directly...
[18:41:44] E/launcher - session not created: This version of ChromeDriver only supports Chrome version 75
  (Driver info: chromedriver=75.0.3770.90 (a6dcaf7e3ec6f70a194cc25e8149475c6590e025-refs/branch-heads/3770@{#1003}),platform=Mac OS X 10.15.3 x86_64)
[18:41:44] E/launcher - SessionNotCreatedError: session not created: This version of ChromeDriver only supports Chrome version 75
  (Driver info: chromedriver=75.0.3770.90 (a6dcaf7e3ec6f70a194cc25e8149475c6590e025-refs/branch-heads/3770@{#1003}),platform=Mac OS X 10.15.3 x86_64)
    at Object.checkLegacyResponse (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/selenium-webdriver/lib/error.js:505:15)
    at parseHttpResponse (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/selenium-webdriver/lib/http.js:509:13)
    at doSend.then.response (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/selenium-webdriver/lib/http.js:440:13)
    at process._tickCallback (internal/process/next_tick.js:68:7)
From: Task: WebDriver.createSession()
    at Function.createSession (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/selenium-webdriver/lib/webdriver.js:777:24)
    at Function.createSession (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/selenium-webdriver/chrome.js:709:29)
    at Direct.getNewDriver (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/lib/driverProviders/direct.ts:90:25)
    at Runner.createBrowser (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/lib/runner.ts:225:39)
    at q.then.then (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/lib/runner.ts:391:27)
    at _fulfilled (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/node_modules/q/q.js:834:54)
    at /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/node_modules/q/q.js:863:30
    at Promise.promise.promiseDispatch (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/node_modules/q/q.js:796:13)
    at /Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/node_modules/q/q.js:556:49
    at runSingle (/Users/mavogel/Development/abc/boilerplates/node-nest-starter-20200403/frontend/node_modules/protractor/node_modules/q/q.js:137:13)
[18:41:44] E/launcher - Process exited with error code 199
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ngx-admin-bundle@3.20.2020 e2e: `ng e2e`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ngx-admin-bundle@3.20.2020 e2e script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /U
```

**Comment by Pakoke** [comment]
@mavogel, did you apply some workaround on this?

**Comment by mavogel** [comment]
@Pakoke unfortunately not. I bought the bundle and expected those commands to work. 

It seems they stripped out several files when selling the product. Furthermore, I'd happy to help set up a CI pipeline for this to ensure all those commands work out of the box. I already inquired the support to fix this but there was no answer yet.

**Comment by velasquezjeisson** [comment]
Hello @mavogel , did you have a solution?
Thanks

**Comment by mavogel** [comment]
Not until now. Maybe @valentinkononov can help on this as he seems one of the maintainers :)

**Comment by valentinkononov** [comment]
@mavogel I've notified people involved in it. Unfortunately, I'm not part of the project now.

**Comment by Cha11enger** [comment]
i am getting the same error is there any way to fix it...?

**Comment by mavogel** [comment]
@mishkolesnikov can you provide help here?

> Furthermore, I'd happy to help set up a CI pipeline for this to ensure all those commands work out of the box

I'm happy to help set up Github actions for this :)
